airbyte-webapp/src/components/connection/CatalogTree/next/StreamPathSelect.module.scss

```diff
@@ -4,6 +4,10 @@
   white-space: nowrap;
 }
 
-.pillSelect {
+.streamPathSelect {
```

Comment:
not an essential piece of this PR, but while digging around in the rows, I saw that the streamPathSelect did not stay the full width of its column like on Figma.  Adjusted that.